modules.go

```diff
@@ -2,10 +2,7 @@ package main
 
 import (
 	_ "github.com/gliderlabs/logspout/adapters/raw"
-	_ "github.com/gliderlabs/logspout/adapters/syslog"
 	_ "github.com/gliderlabs/logspout/httpstream"
 	_ "github.com/gliderlabs/logspout/routesapi"
-	_ "github.com/gliderlabs/logspout/transports/tcp"
-	_ "github.com/gliderlabs/logspout/transports/udp"
-	_ "github.com/logdna/logspout/logdna"
+	_ "github.com/smusali/logspout/logdna"
```

Comment:
Should this be changed back to '_ "github.com/**logdna**/logspout/logdna"' instead of '_ "github.com/**smusali**/logspout/logdna"'?

Reply:
Yup - I'll update them back once all looks good to be merged - for testing, I need `smusali/logspout` for now